asyncapi/json-schemas/operational/assignment/omit/request/common/assignment-omit-journey-call.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$id": "https://schemas.ruter.no/adt/ota/api/v3.0/operational/assignment/omit/request/common/assignment-omit-journey-call.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "AssignmentOmitJourneyCall",
+  "description": "Call used defining calls in a journey",
```

Comment:
could something like "A call defines the specific stops (calls) that a vehicle will make during a journey." be more easier to understand?